host/3.0/buster/amd64/powershell/powershell7.Dockerfile

```diff
@@ -0,0 +1,54 @@
+# Build the runtime from source
+ARG HOST_VERSION=3.0.14358
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1 AS runtime-image
+ARG HOST_VERSION
+
+ENV PublishWithAspNetCoreTargetManifest false
+
+# Build WebJobs.Script.WebHost from source
+RUN BUILD_NUMBER=$(echo ${HOST_VERSION} | cut -d'.' -f 3) && \
+    git clone --branch v${HOST_VERSION} https://github.com/Azure/azure-functions-host /src/azure-functions-host && \
+    cd /src/azure-functions-host && \
+    HOST_COMMIT=$(git rev-list -1 HEAD) && \
+    dotnet publish -v q /p:BuildNumber=$BUILD_NUMBER /p:CommitHash=$HOST_COMMIT src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj -c Release --output /azure-functions-host --runtime linux-x64 && \
+    mv /azure-functions-host/workers /workers && mkdir /azure-functions-host/workers && \
+    rm -rf /root/.local /root/.nuget /src
+
+# Install additional tools needed
+RUN apt-get update && \
+    apt-get install -y gnupg wget unzip jq
+
+# Install just bundle version 2
```

Comment:
I don't think we should get rid of Bundles v1 at this point. Bundles v2 can contain breaking change (this is why we keep releasing v1.* and v2.* separately), so we would unnecessarily break the users trying to migrate from PS6 to PS7, and they will be forced to perform one more migration at the same time. I think we should keep v1 for now, this will also be consistent with the images for other languages.